README.md

```diff
@@ -1047,6 +1047,20 @@ cmp = true
 </tr>
 <!-- nvim-cmp -->
 
+<!-- nvim-copilot-vim -->
+</tr>
+<tr>
+<td> <a href="https://github.com/github/copilot.vim">copilot.vim</a> </td>
+<td>
+
+```lua
+copilot_vim = true,
```

Comment:
Should be set to `false` since it shouldn't be enabled by default as explained below.

```suggestion
copilot_vim = false,
```